examples/runner.py

```diff
@@ -24,6 +24,12 @@
 ZSEQUENCER_SYNC_INTERVAL: float = 0.01
 ZSEQUENCER_FINALIZATION_TIME_BORDER: int = 10
 ZSEQUENCER_SIGNATURES_AGGREGATION_TIMEOUT = 5
+ZSEQUENCER_FETCH_APPS_AND_NODES_INTERVAL = 1
+ZSEQUENCER_SUBGRAPH_URL='https://api.studio.thegraph.com/query/85556/bls_apk_registry/version/latest'
+ZSEQUENCER_RPC_NODE='https://holesky.infura.io/v3/ba262008fcdc4324b86692d0851dd33a'
+ZSEQUENCER_REGISTRY_COORDINATOR='0xa19A698e512240Fe51b8FBf8c3BD86EEC3306379'
+ZSEQUENCER_OPERATOR_STATE_RETRIEVER='0x8ce7f30F2c11e61AEBbAaD3938E201AE33332950'
```

Comment:
that example network is running via local nodes.json file and setting these variables are not required for